recipes/pymumps/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pymumps" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+  
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/santiagoropb/pymumps/archive/v{{ version }}.tar.gz
+  sha256: dddfc13eb6af56a9bcd8e42fd42c3293dc50fd155fe3847c3dd84a403edba8ba
+
+build:
+  number: 1000
+  script: python setup.py install --include-dirs $BUILD_PREFIX/include --library-dirs $BUILD_PREFIX/lib
+  skip: true  # [win]
+  
+requirements:
+  build:
+    - cython
+    - python
+    - mumps
+    - setuptools
+    - mpi4py
+    
+  run:
+    - python
+    - cython
+    - mumps
+    - mpi4py
+    - {{ pin_compatible('numpy') }}
+    - setuptools
```

Comment:
Is this needed as a runtime dependency?

Reply:
@mariusvniekerk thanks for the help with the recipe. I have removed the ones that are not needed as runtime dependecies